pybamm/parameters/parameter_sets.py

```diff
@@ -93,6 +98,7 @@
     "electrolyte": "lipf6_Marquis2019",
     "experiment": "1C_discharge_from_full_Marquis2019",
     "sei": "example",
+    "mechanical": "lico2_graphite_Ai2020",
```

Comment:
maybe not for this issue, but I think we should decide on what is the best way to add parameters for new phsyics. I think now these parameters are defined as properties of the anode/cathode for the electrodes for the Enertech cell, and also defined as a separate list of "mechanical" parameters. 

I can see why it is useful to have the "mechanical" set so you can quickly add them to other parameter sets (I know we do something similar for SEI), but there is the risk that people might just start using these mechanical parameters with a different set of parameters for a different cell in a way that doesn't make sense. maybe it is safer to put the burden on the user to manually update any missing parameters

TL;DR should we just have parameter sets for a given cell/paper and if some parameters needed for extra physics are missing then it is up to the user to manually add them, or should each new physical effect get its parameters stored in a separate csv?

Reply:
It would be good to have a command like `pybamm.import_parameters_from(parameter_set,parameter_list)` so in a single line you can pass a list of the parameters to import and from which parameter set (and maybe even update the current parameters?). This way the burden is minimal but the user is still aware of which parameters they are using.

We might even have tags for types of parameters, as I mentioned above, which would make this task even easier.

Reply:
I think we should have a parameter set for a given cell rather than separating by physical effect. From what I understand, the definitions of parameters are often implicitly coupled in some way (e.g. how you define OCV affects how you define reaction rate)

Reply:
have removed this line